internal/actions/sync_stack.go

```diff
@@ -190,8 +190,9 @@ func SyncStack(ctx context.Context,
 				"  - deleting merged branch ", colors.UserInput(currentBranch),
 				"\n",
 			)
-			if _, err := repo.Git("switch", "--detach"); err != nil {
-				return errors.Errorf("cannot switch to detached HEAD: %v", err)
+
+			if _, err := repo.CheckoutBranch(&git.CheckoutBranch{Name: br.Parent.Name}); err != nil {
+				return errors.Errorf("cannot checkout to parent branch %s: %v", br.Parent.Name, err)
```

Comment:
You mentioned 

> Instead we should checkout trunk

in https://github.com/aviator-co/av/issues/246

Does "trunk' refer to something different from a 'parent branch' or same?

Reply:
Trunk is typically 'main' or 'master'. It'll be the root of those stacked branches.

Reply:
I see, I'll fix it